internal/gps/lock.go

```diff
@@ -195,6 +196,15 @@ func (lp LockedProject) Packages() []string {
 	return lp.pkgs
 }
 
+func (lp LockedProject) String() string {
+	prj := lp.Ident()
+	rev, _, _ := VersionComponentStrings(lp.Version())
```

Comment:
let's leave `versionPair.String()` out of this, at least for now, and just accept a bit of duplicate code for this particular case.